### PR TITLE
Evd creation

### PR DESCRIPTION
**Pull request by oshratZairi**
1. Add evd_signing_key to the environment variables and use it for evidence creation if the key flag is not provided.
2. Add evd_key_alias to the environment variables and use it, if available, instead of requiring the alias_key flag.

### CODE REVIEW

File: evidence/cli/command_cli.go [hunk 1]
```diff
@@ -70,15 +70,42 @@ func validateCreateEvidenceCommonContext(ctx *components.Context) error {
...
 }
 
+func ensureKeyExists(ctx *components.Context, key string) error {
+	if ctx.IsFlagSet(key) && assertValueProvided(ctx, key) == nil {
+		return nil
+	}
+
+	signingKeyValue, _ := getEnvVariable(evdSigningKey)
+	if signingKeyValue == "" {
+		return errorutils.CheckErrorf("'key' or EVD_KEY must be provided when creating evidence: --%s", key)
```
Comment (lesnerd) [inline evidence/cli/command_cli.go:96]: I dont understand what should I do as a user.
1. Provide a param: `key`
2. Provide a environment variable under the name `EVD_KEY`

File: evidence/cli/command_cli.go [hunk 2]
```diff
@@ -70,15 +70,42 @@ func validateCreateEvidenceCommonContext(ctx *components.Context) error {
...
+	}
+	ctx.AddStringFlag(key, signingKeyValue)
+	return nil
+}
+
+func setKeyAliasIfNeeded(ctx *components.Context, keyAlias string) {
+	evdKeyAliasValue, _ := getEnvVariable(evdKeyAlias)
```
Comment (lesnerd) [inline evidence/cli/command_cli.go:103]: Maybe the keyalias was provided as a parameter and the key was read from the environment variable

Comment (oshratZairi) [reply evidence/cli/command_cli.go:103]: if the key was provided as a param - we won't get here; the code in line 82 checks
if !ctx.IsFlagSet(keyAlias) {
       setKeyAliasIfNeeded(ctx, keyAlias)
}
if the key was provided as param - we do nothing and the param value will be taken. if not - we will check if we have env var - if so - we will use it.

File: evidence/cli/command_cli.go (hunk 1 above)
Comment (oshratZairi) [reply evidence/cli/command_cli.go:96]: I agree. need a better msg for the user. The user must provide: --key OR define env var EVD_SIGNING_KEY. 
do u have a better suggestion for msg?

File: evidence/cli/command_cli.go (hunk 2 above)
Comment (lesnerd) [reply evidence/cli/command_cli.go:103]: Then change the function name, if you call `setKeyAliasIfNeeded` you indeed need alias.
By the way it still not guaranteed to have such thing.

**Comment by lesnerd** [comment]
.

**Review by lesnerd — APPROVED** [review]

**Pull request merged**